app/models/contributor.rb

```diff
@@ -39,11 +39,12 @@ class Contributor < ApplicationRecord
   validates :whats_app_phone_number, phony_plausible: true
   validates :data_processing_consent, acceptance: true, unless: proc { |c| c.editor_guarantees_data_consent }
 
-  validates :email, uniqueness: { case_sensitive: false }, allow_nil: true, 'valid_email_2/email': true
-  validates :threema_id, uniqueness: { case_sensitive: false }, allow_blank: true
-  validates :telegram_id, uniqueness: true, allow_nil: true
-  validates :signal_phone_number, uniqueness: true, allow_nil: true
-  validates :whats_app_phone_number, uniqueness: true, allow_nil: true
+  validates :email, uniqueness: { case_sensitive: false, scope: :organization }, allow_nil: true, 'valid_email_2/email': true
+  validates :threema_id, uniqueness: { case_sensitive: false, scope: :organization }, allow_blank: true
+  validates :telegram_chat_id, uniqueness: { scope: :organization }, allow_nil: true
+  validates :telegram_id, uniqueness: { scope: :organization }, allow_nil: true
+  validates :signal_phone_number, uniqueness: { scope: :organization }, allow_nil: true
+  validates :whats_app_phone_number, uniqueness: { scope: :organization }, allow_nil: true
```

Comment:
```suggestion
  validates :whats_app_phone_number, uniqueness: { scope: :organization_id }, allow_nil: true
```

It seems to work with organization, but I'd consider using the `organization_id` since all the docs I can find on this scoping validation use the `_id`

Reply:
I don't think that it improves something .. but let's get it merged: https://github.com/tactilenews/100eyes/pull/2004/commits/14510e3904cf0f78de52661b8277a7e09b0f4252